pkg/archive/archive_windows.go

```diff
@@ -23,6 +24,17 @@ func CanonicalTarNameForPath(p string) (string, error) {
 	return strings.Replace(p, "\\", "/", -1), nil
 }
 
+// chmodTarEntry is used to adjust the file permissions used in tar header based
+// on the platform the archival is done.
+func chmodTarEntry(perm os.FileMode) os.FileMode {
+	// Clear r/w on grp/others: no precise equivalen of group/others on NTFS.
+	perm &= 0711
```

Comment:
@tiborvass so direct answer to your question is, directory entries are also passed to this method and we already have `x` bit on those from os.Stat, we don't want to clear those bits.